Lesson26/Kubra/script.js

```diff
@@ -28,17 +21,42 @@ function fetchPosts() {
 				paragraph.innerText = post.body;
 				listItem.appendChild(paragraph);
 
+				const editButton = document.createElement('a');
+				editButton.classList.add('button');
```

Comment:
You can add all classes at the same time:
```suggestion
				editButton.classList.add('button button--danger button--edit');
```